analyzers/src/SonarAnalyzer.CSharp.Styling/Rules/LambdaParameterName.cs

```diff
@@ -23,13 +23,13 @@ namespace SonarAnalyzer.Rules.CSharp.Styling;
 [DiagnosticAnalyzer(LanguageNames.CSharp)]
 public sealed class LambdaParameterName : StylingAnalyzer
 {
-    public LambdaParameterName() : base("T0010", "Use 'x' for the lambda parameter name.") { }
+    public LambdaParameterName() : base("T0010", "Use 'x' or '_' for the lambda parameter name.") { }
```

Comment:
I would not change the message. The rule now asks me to use `x` or `_`, so technically it asks me to `_ => _ + 1`.